tv_apps/smart-system/js/app_modal_dialog.js

```diff
@@ -363,6 +417,15 @@
     this.elements[type].classList.remove('visible');
```

Comment:
I don't think we need line 415 since line 401 already remove the blur of activeElement.